miso-web/src/main/webapp/scripts/sample_ajax.js

```diff
@@ -436,7 +436,7 @@ Sample.library = {
     }
   },
   
-  validateLibraryQcs: function () {
+  validateLibraryQcs: function (json) {
```

Comment:
It's a fix for a  previous commit (https://github.com/oicr-gsi/miso-lims/commit/494ae1b6ba551fb651be0ee4b497ec6801d2c93a#diff-dc20b4f617c4dd4cdee5a309a1f97d0aL184) in which I refactored the method and accidentally removed the JSON argument.